front/app/containers/EventsPage/UpcomingEventsViewer.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import EventsViewer from './EventsViewer';
+
+// i18n
+import { injectIntl } from 'utils/cl-intl';
+import { InjectedIntlProps } from 'react-intl';
+import messages from './messages';
+
+interface Props {
+  upcomingEvents: number[];
```

Comment:
I know this is mock data, but wouldn't pass events such a long way. I think they should be fetched inside UpcomingEvents and PastEvents via a parameter in the request.

Reply:
Completely agree, will change this in the next PR when I will hook up everything with the API